images/builder/main.go

```diff
@@ -38,6 +38,42 @@ const (
 	gcsLogsDir   = "/logs"
 )
 
+type Step struct {
+	Name string `yaml:"name"`
+	Args []string
+}
+
+type CloudBuildYAMLFile struct {
+	Steps         []Step `yaml:"steps"`
+	Substitutions map[string]string
+	Images        []string
+}
+
+func getProjectId() (string, error) {
+	cmd := exec.Command("gcloud", "config", "get-value", "project")
```

Comment:
This is an interesting way to get a project's ID in a scope. It seems both fascinating and maybe brittle, but I don't know of a better way to do this. Logging it when it's called seems too noisy to be worthwhile, too.

No notes, I guess